### PR TITLE
text input doc changes

### PR DESCRIPTION
**Pull request by mmenestr**
This would fix issue #1817

### CODE REVIEW

File: packages/v4/src/content/design-guidelines/usage-and-behavior/text-input/text-input.md [hunk 1]
```diff
@@ -9,38 +9,39 @@ Use text input to gather free-form text from a user.
...
 ## Usage
 * Be sure to size input fields appropriately to anticipate the longest likely value (if a maximum character length is known).
 * When a value is too long to be fully displayed in a text field, use [truncation](/design-guidelines/content/grammar-and-terminology#truncation).
 * Provide a [text area](/documentation/react/components/textarea) for input anticipated to be longer than a single line.
-
-![Text area](./img/textarea.png)
-
-* Pre-populate fields with known values when possible, e.g. a default IP address.
+* Pre-populate fields with known values when possible, like a default IP address, for example.
 * When presented in a form, the initial input field is usually the first input field required. The primary input field should receive focus when the form is presented to the user.
```
Comment (mcarrano) [inline packages/v4/src/content/design-guidelines/usage-and-behavior/text-input/text-input.md:24]: Not sure that I understand what this is saying. Perhaps you can reword?

File: packages/v4/src/content/design-guidelines/usage-and-behavior/text-input/text-input.md [hunk 2]
```diff
@@ -9,38 +9,39 @@ Use text input to gather free-form text from a user.
...
 
 ### Help
```
Comment (mcarrano) [inline packages/v4/src/content/design-guidelines/usage-and-behavior/text-input/text-input.md:26]: This page talks further about how and when to use different forms of on-screen help: https://www.patternfly.org/v4/design-guidelines/usage-and-behavior/embedded-assistance  We should reference and link this.

File: packages/v4/src/content/design-guidelines/usage-and-behavior/text-input/text-input.md [hunk 3]
```diff
@@ -9,38 +9,39 @@ Use text input to gather free-form text from a user.
...
-Helper text is text below a form field that helps a user provide the right information, like Enter a unique name.
 
-**Placeholder text**
-Placeholder text is text inside a form field that provides an example of the required or recommended format for text input. For example, placeholder text for a date might be e.g. DD/MM/YYYY.
+[Helper text](/design-guidelines/usage-and-behavior/forms#helper-text) is text below a form field that helps a user provide the right information, like "Enter a unique name".
+
+
+**Popovers**
+
+[Popovers](/design-guidelines/usage-and-behavior/forms#popovers) provide context around a text or input field. Popovers are represented with an unfilled question icon and displayed when the user clicks on the icon.
```
Comment (mcarrano) [inline packages/v4/src/content/design-guidelines/usage-and-behavior/text-input/text-input.md:38]: Include the exact icon name?

File: packages/v4/src/content/design-guidelines/usage-and-behavior/text-input/text-input.md [hunk 4]
```diff
@@ -9,38 +9,39 @@ Use text input to gather free-form text from a user.
...
 
-For more information about how to provide help content, see  the [forms design guidelines](/design-guidelines/usage-and-behavior/forms#provide-the-right-kind-of-help).
+For more information about how to provide help content, see  the [forms design guidelines](/design-guidelines/usage-and-behavior/forms#user-help).
 
 ### Errors and validation
 Wherever possible, use inline errors to notify users that something has gone wrong. Inline errors help should be short and help users quickly identify and understand what they need to fix and why.
```
Comment (mcarrano) [inline packages/v4/src/content/design-guidelines/usage-and-behavior/text-input/text-input.md:43]: Change "something has gone wrong" to "there is a problem with the user's input"?

File: packages/v4/src/content/design-guidelines/usage-and-behavior/text-input/text-input.md (hunk 1 above)
Comment (mmenestr) [reply packages/v4/src/content/design-guidelines/usage-and-behavior/text-input/text-input.md:24]: Note to self: remove that bottom bullet

File: packages/v4/src/content/design-guidelines/usage-and-behavior/text-input/text-input.md (hunk 3 above)
Comment (mmenestr) [reply packages/v4/src/content/design-guidelines/usage-and-behavior/text-input/text-input.md:38]: Note to self: Ask Gina what the correct icon name is

**Comment by mmenestr** [comment]
@mcarrano just pushed some new changes to this text input documentation as well! I just added the input group documentation in here like we had discussed, and the previous changes were just general updates to the doc.

**Pull request merged**